number.py

```diff
@@ -3,7 +3,7 @@
 # Give the list as static input and store it in a variable.
 list1 = [4, 1, 4, 145]
 # Loop in the above given list using For Loop.
-print("The Strong Numbers in a given List are :")
```

Comment:
This is aTrailingwhitespace

```suggestion
{suggested_changes_content_here}
```

 from bot